doc/source/cluster/running-applications/job-submission/openapi.yml

```diff
@@ -0,0 +1,471 @@
+openapi: 3.1.0
```

Comment:
is this file manually written? Did we consider switching the jobs api to use fast-api which has the ability to automatically generate openapi spec from the route definitions?

@rkooo567 and I have been chatting about doing a fast-api migration in the next 6 months so that we can have openapi specs for all of our APIs. If we are to do that, there's a chance that this spec will be slightly modified because it's generated instead of manually written. Will that be disruptive to users?

Reply:
@alanwguo Yeah, migrating Jobs to FastAPI makes sense as the long term plan, we just didn't have time to do it for Ray 2.2.  I think by marking the OpenAPI section of the doc as "Beta" as it currently is marked this PR, it should be okay if there are slight modifications in the future. The underlying API is stable and once we migrate to OpenAPI we can mark the OpenAPI doc as "stable" as well.